content/en/profiler/guide/save-cpu-in-production-with-go-pgo.md

```diff
@@ -37,31 +37,31 @@ To enable PGO using the `datadog-pgo` tool:
 
 1. Create a dedicated API key and an application key scoped to at least `continuous_profiler_pgo_read` as described in [API and Application Keys][5].
 2. Set `DD_API_KEY`, `DD_APP_KEY`, and `DD_SITE` with the environment secret mechanism of your CI provider.
-3. Run `datadog-pgo` before your build step.  
+3. Run `datadog-pgo` before your build step.
    For example, for a service `foo` that runs in `prod` and has its main package in `./cmd/foo`, you should add this step:
 
    ```
-   go run github.com/DataDog/datadog-pgo@latest 'service:foo env:prod' ./cmd/foo/default.pgo
+   go run github.com/DataDog/datadog-pgo@latest "service:foo env:prod" ./cmd/foo/default.pgo
```

Comment:
This is the change I'm trying to make. The other changes were made by the local tooling that reformats the docs. Previous PRs might have been submitted from the GH UI and were not formatted properly.